plugins/proxy-backend/config.d.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface Config {
+  /**
+   * A list of forwarding-proxies. Each key is a route to match,
+   * below the prefix that the proxy plugin is mounted on. It must
+   * start with a '/'.
+   */
+  proxy: { [key: string]: string | ProxyConfig };
+}
+
+interface ProxyConfig {
```

Comment:
While this works it creates a global definition that is shared with all other schema definitions. It was really an oversight not to forbid it from the beginning, and gonna go ahead and deprecate it. Let's move this to an inline literal type for now.

Reply:
I inlined the interface.